### PR TITLE
Drop the extra ukranian host

### PR DESCRIPTION
**Pull request by jakubgs**
It's causing a lot of issues in infra automation since we don't have a consul setup for that DC, and the Google Cloud hosts are available in russia so removing it should be fine.

### CODE REVIEW

**Comment by adambabik** [comment]
Waiting for a status-go build to also update status-go version: https://jenkins.status.im/job/status-go/job/status-go-manual/247/

<blockquote></blockquote>

**Review by pedropombeiro — APPROVED** [review]
LGTM, once we update the Go version

**Review by cammellos — APPROVED** [review]
had problems with this particular mailserver this morning :)

**Comment by cammellos** [comment]
actually, this might be problematic for those users that are already connected to mailserver `g` as we don't reset the preference, did we ever release the new mailservers?

**Comment by adambabik** [comment]
@cammellos I don't think so. This is the first time we are releasing the new cluster.

**Comment by adambabik** [comment]
Mail Server `g` was introduce in this PR: https://github.com/status-im/status-react/pull/4642

**Comment by adambabik** [comment]
Also, it's kinda hard to follow which commit made to which release. We really should update tags accordingly...

**Comment by cammellos** [comment]
@adambabik thanks for looking into this, that's a good shout about tagging

**Comment by pedropombeiro** [comment]
There's this bounty I've created a few days ago, would be useful in that scenario: https://github.com/status-im/status-react/issues/4643

**Comment by lukaszfryc** [comment]
Can we migrate users that have used mailserver `g`? Offline inboxing does not work for them after the upgrade. Looks like `g` is still selected but null:  


@cammellos maybe you could help?

**Comment by cammellos** [comment]
@lukaszfryc I am working on a related issue that should fix it, should be ready fairly soon https://github.com/status-im/status-react/issues/4643, but otherwise I believe this should only impact nightly->nightly upgrades, as the new mailservers have never been released, correct me if I am  wrong.

**Comment by jakubgs** [comment]
I've merged this using the `merge-pr` script but it seems it didn't close this PR so I'm doing it by hand.

**Comment by lukaszfryc** [comment]
@cammellos makes sense. Yes, it has not been released yet.

**Comment by divan** [comment]
@jakubgs is there anything related to host itself that causing the problems? Or it's just infra tooling doesn't work well with custom hosts?
